clients/cloud/rust/README.md

```diff
@@ -0,0 +1,61 @@
+# Overview
+
+Produce messages to produce and consume messages from [Confluent Cloud](https://www.confluent.io/confluent-cloud/) using the [rust-rdkafka client for Apache Kafka](https://github.com/fede1024/rust-rdkafka).
+
+# Prerequisites
+* [Rust client for Apache Kafka](https://github.com/fede1024/rust-rdkafka#installation) installed on your machine
+* Access to a [Confluent Cloud](https://www.confluent.io/confluent-cloud/) cluster
+* Local file with configuration parameters to connect to your Confluent Cloud instance ([how do I find those?](https://docs.confluent.io/current/cloud/using/config-client.html#librdkafka-based-c-clients)). Format the file as follows:
+
+```bash
+$ cat ~/.ccloud/example.config
+bootstrap.servers=<broker-1,broker-2,broker-3>
+sasl.username=<api-key-id>
+sasl.password=<secret-access-key>
+```
+
+# Example 1: Hello World!
+In this example, the producer writes Kafka data to a topic in Confluent Cloud. 
```

Comment:
"writes data to a Kafka topic"

Reply:
copy/paste from other client examples - I left it as is for uniformity

I guess I could change in all client example readme's